express-post-json/index.js

```diff
@@ -0,0 +1,28 @@
+const express = require('express');
+const app = express();
+var nextId = 1;
+const grades = {};
+
+function returnArr() {
+  const gradesArr = [];
+  for (const id in grades) {
+    gradesArr.push(grades[id]);
+  }
+  return gradesArr;
+}
+
+app.get('/api/grades', function (req, res) {
+  res.json(returnArr());
+});
+
+app.use(express.json());
+
+app.post('/api/grades', function (req, res) {
+  grades[nextId] = req.body;
+  grades[nextId].id = nextId;
+  nextId++;
+  res.sendStatus(201);
```

Comment:
Don't forget to send a copy of the newly-created grade back to the client:

---

![image](https://user-images.githubusercontent.com/16143771/142139023-4acde00d-68dc-4c76-80dc-039d3173729c.png)